docs/usage.rst

```diff
@@ -385,6 +385,41 @@ XPath specification.
 .. _Location Paths: https://www.w3.org/TR/xpath#location-paths
 
 
+Removing elements
+-----------------
+
+If for any reason you need to remove elements based on a Selector or
+a SelectorList, you can do it with the ``remove()`` method, available for both
+classes.
+
+.. warning:: this is a destructive action and cannot be undone. The original
```

Comment:
I’m not sure the warning is really necessary as long as the documentation is clear, because I think it is unlikely that a selector will be used in a scenario where loosing data on the selector will result in the data becoming unrecoverable (users will usually still have access to the text passed to the selector).

That said, if you really want to include the warning, I would ask for a minor change:

```suggestion
.. warning:: This is a destructive action and cannot be undone. The original
```